mobile-app/app/screens/WalletNavigator/screens/CreateWallet/OnboardingNetworkSelectScreen.tsx

```diff
@@ -1,30 +1,152 @@
-import { ThemedSectionTitle } from '@components/themed'
-import { getEnvironment } from '@environment'
-import { RowNetworkItem } from '@components/RowNetworkItem'
+import { RootState } from '@store'
+import dayjs from 'dayjs'
+import { useSelector } from 'react-redux'
+import NumberFormat from 'react-number-format'
+import { TouchableOpacity, Linking } from 'react-native'
+import { View } from '@components/index'
+import { ThemedIcon, ThemedTextV2, ThemedViewV2, ThemedSectionTitleV2 } from '@components/themed'
+import { tailwind } from '@tailwind'
 import { translate } from '@translations'
-import { View } from 'react-native'
+import { useDeFiScanContext } from '@shared-contexts/DeFiScanContext'
+import { TextRowV2 } from '@components/TextRowV2'
+import { NumberRowV2 } from '@components/NumberRowV2'
+import { getEnvironment } from '@environment'
 import { getReleaseChannel } from '@api/releaseChannel'
+import { NetworkItemRowV2 } from '@components/NetworkItemRowV2'
 
 export function OnboardingNetworkSelectScreen (): JSX.Element {
   const networks = getEnvironment(getReleaseChannel()).networks
+  const {
+    count: blockCount,
+    masternodeCount,
+    lastSuccessfulSync
+  } = useSelector((state: RootState) => state.block)
+  const syncFormattedDate = (lastSuccessfulSync != null) ? dayjs(lastSuccessfulSync).format('lll') : ''
 
   return (
-    <View testID='onboarding_network_selection_screen'>
```

Comment:
Found a test still using this ^
<img width="438" alt="image" src="https://user-images.githubusercontent.com/13292662/179700941-5a127929-9169-42fd-8bc2-b5ba838d12a5.png">